CHANGELOG.md

```diff
@@ -5,6 +5,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.1] - 2025-01-19
+
+This release enhances compatibility between litep2p and libp2p by backporting the latest Yamux updates. Additionally, it includes various improvements and fixes to boost the stability and performance of the WebSocket stream and the multistream-select protocol.
```

Comment:
> backporting the latest Yamux updates

What does this mean? Was anything backported? I thought we just use the upstream version and thats it.

Reply:
Yep that's right, I've rephrased to highlight the upstream version update 🙏